envergo/contrib/management/commands/backup_s3_bucket.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management import BaseCommand
+
+from envergo.contrib.tasks import execute_s3_backup
+
+
+class Command(BaseCommand):
+    help = "backup all the files in the s3 bucket to a cold storage."
+
+    def handle(self, *args, **options):
+        execute_s3_backup()
```

Comment:
Sauf si je me trompe, cette tâche sera executée en synchrone. C'est voulu ?

Sinon il faut peut-être :

```suggestion
        execute_s3_backup.delay()
```